### PR TITLE
chore: update version 6.0.7

### PR DESCRIPTION
**Pull request by jeffshuai**
update version 6.0.7

Log: update version 6.0.7

### CODE REVIEW

**Comment by hundundadi** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Comment by myk1343** [comment]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Review by pengfeixx — APPROVED** [review]

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+deepin-system-monitor (6.0.7) UNRELEASED; urgency=medium
```
Comment (wineee) [inline debian/changelog:1]: UNRELEASED 不会打 tag，应该是 unstable 吧？

**Pull request merged**